packages/workbox-build/package.json

```diff
@@ -39,7 +39,7 @@
   },
   "main": "build/index.js",
   "scripts": {
-    "build": "gulp build:update-cdn-details && gulp build-packages --package workbox-build",
+    "build": "gulp build-packages --package workbox-build",
```

Comment:
This change is likely to be a problem. There is a subtle dance around CDN updates that is sadly fragile and I'm not sure this will have the same output.

Reply:
I'll split this task into it's own file and expose it to the CLI and put this back.

Reply:
I'm fine with whatever. I don't have a job right now so I'm just looking through large uses of gulp 4 to learn people's patterns and make suggestions. Let me know what you'd like to do.